lib/ConfigUtils.js

```diff
@@ -10,7 +10,7 @@ exports.propsToOptions = function (props) {
         'resizableColumns', 'resizableRows', 'autoResize', 'tooltips', 'tooltipsHeader', 'tooltipGenerationMode',
         'initialSort', 'initialFilter', 'footerElement', 'index', 'keybindings', 'clipboard', 'clipboardCopyStyled',
         'clipboardCopySelector', 'clipboardCopyFormatter', 'clipboardCopyHeader', 'clipboardPasteParser',
-        'clipboardPasteAction', 'rowFormatter', 'placeholder'];
+        'clipboardPasteAction', 'rowFormatter', 'placeholder', 'selectable'];
```

Comment:
Thanks for your PR!
We need to make change to ConfigUtils.tsx. This ConfigUtils.js is just the output after compiling.